packages/tables/src/examples/draggable-table.md

```diff
@@ -97,10 +97,7 @@ class DraggableExample extends React.Component {
           <Droppable droppableId="droppable">
             {(provided, droppableSnapshot) => {
               return (
-                <Body
-                  innerRef={provided.innerRef}
-                  isDraggingOver={droppableSnapshot.isDraggingOver}
-                >
+                <Body ref={provided.innerRef} isDraggingOver={droppableSnapshot.isDraggingOver}>
```

Comment:
Is `<Droppable />` still calling the prop `innerRef`?

Reply:
`innerRef` is the value provided by the render-prop in `<Droppable />` so nothing on our end. It's applied to the correct `ref` so we should be good.